src/idvc/io.py

```diff
@@ -551,9 +732,18 @@ def loadTif(*args, **kwargs):
 
         image_info['sampled'] = False
 
-    # this is dangerous as reader might not be defined!!!
-    vol_bit_depth = reader.GetBytesPerElement() * bits_per_byte
-
+    if image_data.dtype in ["int8", "uint8"]:
```

Comment:
edit method to get the type of the tiff images

Reply:
These can be accessed with the reader with the following methods 

https://github.com/TomographicImaging/CILViewer/blob/191367663679377fdcbebc701bbb30be402c6bbc/Wrappers/Python/ccpi/viewer/utils/conversion.py#L416-L418

So, something like this
```suggestion
    import vtk
    if reader.GetOutputVTKType() in [vtk.VTK_SIGNED_CHAR, vtk.VTK_UNSIGNED_CHAR]:
```

Reply:
applied